jepostule/queue/handlers/franz.py

```diff
@@ -48,13 +48,18 @@ def __init__(self, topic):
             self.topic,
             bootstrap_servers=settings.KAFKA_BOOTSTRAP_SERVERS,
             group_id=self.GROUP_ID,
-            enable_auto_commit=False,
+            enable_auto_commit=True,
             consumer_timeout_ms=self.TIMEOUT_MS,
+            auto_offset_reset='earliest',
```

Comment:
J'ai changé deux paramètres @vgrange : 
- enable_auto_commit à True : ainsi, le Consumer envoie automatiquement à Kafka l'endroit dans lequel il est dans la file et le suivant peut reprendre là où il s'est arrêté.
- auto_offset_reset='earliest' : la valeur par défaut est 'latest'. C'était un essai pour que le Consumer lise les messages depuis le début mais ça n'a pas vraiment fonctionné. Enfin je crois.
La doc : https://kafka-python.readthedocs.io/en/master/apidoc/KafkaConsumer.html

Reply:
`enable_auto_commit (bool) – If True , the consumer’s offset will be periodically committed in the background. Default: True.` => True est la valeur par défaut, on demandera au doc pourquoi il l'avait changée

`consumer_timeout_ms (int) – number of milliseconds to block during message iteration before raising StopIteration (i.e., ending the iterator). Default block forever [float(‘inf’)].` => On l'a descendu de infini à une valeur très aggressive de 3s. C'est un bon suspect, je vais creuser.

Reply:
Ce post éclaircit beaucoup `consumer_timeout_ms`. https://stackoverflow.com/questions/45144298/how-to-close-kafka-consumer-once-all-messages-are-consumed

Le descendre de default à une valeur finie peut expliquer que le consumer se déconnecte dans certaines conditions (sur lbbaux1 surchargé, au relancement du container du consumer). Ca matche bien ce que j'ai vu en prod (container du consumer up mais du point de vue de kafka aucun consumer connecté).

Reply:
https://stackoverflow.com/questions/45144298/how-to-close-kafka-consumer-once-all-messages-are-consumed

`500 miliseconds is not enough time to join the group. You can either set the timeout to say 5mins` => notre 3000ms couplé avec lbbaux1 pourrait expliquer que notre consumer ait du mal à "joindre son groupe" et abandonne précocément.

Reply:
Ces deux changements me paraissent être des erreurs. On en discute ?

Reply:
Oui, ce sont des erreurs ! 
Je prépare une PR pour réparer ça.